### PR TITLE
feat: use jwt token to get extra user information

### PR DESCRIPTION
**Pull request by Ian2012**
## Description
This PR uses a JWT token to get extra user information from the superset-sso.

## Testing instructions

1. Run `tutor local do init --limit superset` to create the necessary oauth application.
2. Login to superset.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: tutorsuperset/templates/superset/apps/pythonpath/openedx_sso_security_manager.py [hunk 1]
```diff
@@ -116,39 +148,19 @@ def get_courses(self, username, permission="staff", next_url=None):
...
-            username=username,
-            is_superuser=False,
-            is_staff=False,
-        )
-
-    cursor.close()
-    cxn.close()
-    return user_access
 
+    claims = jwt.decode(jwt_token, '')
```
Comment (Ian2012) [inline tutorsuperset/templates/superset/apps/pythonpath/openedx_sso_security_manager.py:159]: How can I implement the decoding part?

@bmtcril

Comment (pomegranited) [reply tutorsuperset/templates/superset/apps/pythonpath/openedx_sso_security_manager.py:159]: @Ian2012  Does [this code](https://github.com/openedx/auth-backends/blob/d0ad4d7912c8ae70608bdc8f6e1a0d070d8cd520/auth_backends/backends.py#L100-L106) help?

**Comment by itsjeyd** [comment]
@Ian2012 Thank you for your contribution! Please let us know when it is ready for review.

File: tutorsuperset/templates/superset/apps/pythonpath/openedx_sso_security_manager.py (hunk 1 above)
Comment (Ian2012) [reply tutorsuperset/templates/superset/apps/pythonpath/openedx_sso_security_manager.py:159]: Yes, it helps. but the current code is requesting a JWT for the superset user (the user associated with the application), and not for the current user.

**Comment by openedx-webhooks** [comment]
@Ian2012 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/tutor-contrib-superset&entry.867055334=https://github.com/openedx/tutor-contrib-superset/pull/26&entry.1484655318=https://github.com/Ian2012&entry.752974735=2023-04-25+20:07&entry.1917517419=2023-04-28+16:53&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**